tools/linter_lib/custom_check.py

```diff
@@ -446,23 +446,39 @@ def build_custom_checkers(by_lang):
     ] + whitespace_rules[0:1]  # type: RuleList
     css_rules = cast(RuleList, [
         {'pattern': '^[^:]*:\S[^:]*;$',
-         'description': "Missing whitespace after : in CSS"},
+         'description': "Missing whitespace after : in CSS",
+         'good_lines': ["background-color: white;", "text-size: 16px;"],
+         'bad_lines': ["background-color:white;", "text-size:16px;"]},
         {'pattern': '[a-z]{',
-         'description': "Missing whitespace before '{' in CSS."},
+         'description': "Missing whitespace before '{' in CSS.",
+         'good_lines': ["input {", "body {"],
+         'bad_lines': ["input{", "body{"]},
         {'pattern': 'https://',
-         'description': "Zulip CSS should have no dependencies on external resources"},
+         'description': "Zulip CSS should have no dependencies on external resources",
+         'good_lines': ["background: url(\"background.jpg\")"],
+         'bad_lines': ["background: url(\"https://example.com/background.jpg\")"]},
         {'pattern': '^[ ][ ][a-zA-Z0-9]',
          'description': "Incorrect 2-space indentation in CSS",
          'exclude': set(['static/third/thirdparty-fonts.css']),
-         'strip': '\n'},
+         'strip': '\n',
+         'good_lines': [" color: white;"],
```

Comment:
Good is 4 spaces -- 1-space is wrong (just not tested for by this rule, because nobody does that).